doc/appConfiguration.md

```diff
@@ -16,7 +16,7 @@ application.
   CUMULATED_LOGS)
 - **SUPPORT_URL** is the url (expressed as a string) of the support page that can be get in the help menu symbolysed by a "?".
   If this value is set to null, the item is disabled and not displayed.
-  - **DOCUMENTION_URL** is the url (expressed as a string) of the documentation.
+- **DOCUMENTION_URL** is the relative path to the documentation file from the 'public' folder, located at the project's root. _In this example the default doc.pdf file is located directly in the 'public' folder, so the DOCUMENTATION_URL path value is 'doc.pdf'_.
```

Comment:
Could you also fix the typo in `DOCUMENTION_URL` please ?

Reply:
yes, didn't see that, thanks